samples/ChainedSkills/cs/ChainedSkillsSample_UWP/MainPage.xaml.cs

```diff
@@ -0,0 +1,617 @@
+                    //{
+                    //    objectDetections = objectDetections.Where(det => m_objectKinds.Contains(det.Kind)).ToList();
+                    //}
+
+                    //// Update displayed results
+                    //// FIXME
+                    //m_renderer.Render(objectDetections);
+
+                    // Render results
+                    m_renderer.ClearCanvas();
+                    m_renderer.RenderTrackerResults(m_trackerHistories, true);
+
+                    // Update the displayed performance text
+                    UIPerfTextBlock.Text = $"bind: {bindTime.ToString("F2")}ms, eval: {evalTime.ToString("F2")}ms";
+                }
+                catch (TaskCanceledException)
+                {
+                    // no-op: we expect this exception when we change media sources
+                    // and can safely ignore/continue
+                }
+                catch (Exception ex)
+                {
+                    NotifyUser($"Exception while rendering results: {ex.Message}");
+                }
+            });
+        }
+
+        /// <summary>
+        /// Configure an IFrameSource from a StorageFile or MediaCapture instance to produce optionally a specified format of frame
+        /// </summary>
+        /// <param name="source"></param>
+        /// <param name="inputImageDescriptor"></param>
+        /// <returns></returns>
+        private async Task ConfigureFrameSourceAsync(object source, ISkillFeatureImageDescriptor inputImageDescriptor = null)
+        {
+            await m_lock.WaitAsync();
+            {
+                // Reset bitmap rendering component
+                UIProcessedPreview.Source = null;
+                m_renderTargetFrame = null;
+                m_processedBitmapSource = new SoftwareBitmapSource();
+                UIProcessedPreview.Source = m_processedBitmapSource;
+
+                // Clean up previous frame source
+                if (m_frameSource != null)
+                {
+                    m_frameSource.FrameArrived -= FrameSource_FrameAvailable;
+                    var disposableFrameSource = m_frameSource as IDisposable;
+                    if (disposableFrameSource != null)
+                    {
+                        // Lock disposal based on frame source consumers
+                        disposableFrameSource.Dispose();
+                    }
+                }
+
+                // Create new frame source and register a callback if the source fails along the way
+                m_frameSource = await FrameSourceFactory.CreateFrameSourceAsync(
+                    source, 
+                    (sender, message) => 
+                    {
+                        NotifyUser(message);
+                    },
+                    inputImageDescriptor);
+
+                // TODO: Workaround for a bug in ObjectDetectorBinding when binding consecutively VideoFrames with Direct3DSurface and SoftwareBitmap
```

Comment:
>Workaround for a bug in ObjectDetectorBinding when binding consecutively VideoFrames [](start = 25, length = 84)

is this still the case?

Reply:
I couldn't find an internal bug matching this (must've forgotten to file one) and it doesn't appear to repro on 19H1, so I think no?